.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
Can we do it with lint instead?

Reply:
This is just the config to match the current formatting styles, but prettier also supports formatting Angular's control flow (https://prettier.io/blog/2023/11/13/3.1.0.html#angular)

Reply:
I prefer not to add another dependency if it is not required...

Reply:
okay